ignite/services/chain/generate.go

```diff
@@ -55,39 +55,49 @@ func GenerateOpenAPI() GenerateTarget {
 	}
 }
 
-func (c *Chain) generateAll(ctx context.Context, cacheStorage cache.Storage) error {
+// generateFromConfig makes code generation from proto files from the given config
+func (c *Chain) generateFromConfig(ctx context.Context, cacheStorage cache.Storage, generateGo bool) error {
 	conf, err := c.Config()
 	if err != nil {
 		return err
 	}
 
-	var additionalTargets []GenerateTarget
+	var targets []GenerateTarget
 
+	// generate Go pb files
+	if generateGo {
```

Comment:
We should skip all proto related code generation when this flag is provided, not only go

Reply:
Why? you may have your own way to build proto to Go but still want to generate proto for vuex, etc...

Reply:
We also have --proto-all-modules flag, when you use it enables code generation for all modules in general and not just Go, but also for Vuex and others.

If we use --skip-proto name to only skip Go codegen it wouldn't be ideal because it also implies Vuex and others and not just Go, because of our name convention. 

I suggest to skip all of the code generation and in the future we can provide sub flags or a different configuration format in config.yml to allow Go, Vuex, etc. specific configuration.